### PR TITLE
fix: parameterize titles correctly

### PR DESCRIPTION
**Pull request by suddjian**
### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset/views/core.py [hunk 1]
```diff
@@ -837,7 +837,7 @@ def explore(  # pylint: disable=too-many-locals,too-many-return-statements,too-m
                 bootstrap_data, default=utils.pessimistic_json_iso_dttm_ser
             ),
             entry="explore",
-            title=title,
+            title=title.__str__(),
```
Comment (nytai) [inline superset/views/core.py:840]: wait, so `title` is a instance of LazyString? Seems like it should just be a string here

Comment (suddjian) [reply superset/views/core.py:840]: Title is an instance of lazystring in certain cases. The call to `__str__()` evaluates it to a string.

Comment (nytai) [reply superset/views/core.py:840]: i see now, it's `title = _("Explore - %(table)s", table=table_name)` that's the problematic code path

**Review by nytai — APPROVED** [review]

File: superset/views/core.py (hunk 1 above)
Comment (mistercrunch) [reply superset/views/core.py:840]: any reason to use `title.__str__()` over `str(title)`?

**Pull request merged**